src/services/db/GitHubService.ts

```diff
@@ -241,10 +235,7 @@ export default class GitHubService {
 
   async readMedia(
     sessionData: UserWithSiteSessionData,
-    {
-      fileSha,
-      branchName = STAGING_BRANCH,
-    }: { fileSha: any; branchName?: string }
+    { fileSha }: { fileSha: any; branchName?: string }
```

Comment:
nit: wrong typing - remove `branchName`! could we also stricten `fileSha` to `string`?

Reply:
good catch, removed